### DISCUSSION TITLE
Pimoroni Pico Plus 2

### DISCUSSION THREAD

**Discussion by ce4aser** [micropython/micropython#16077] [General]
Hello, where is support for Pimoroni Pico Plus 2?
 
https://github.com/pimoroni/pimoroni-pico-rp2350

I was  add this board to source code but compiling have error. 

Why not official micropython not support boards?

**Comment by robert-hh** [comment]
You should be able to use the Generic Pico 2 firmware.

**Reply by ce4aser** [reply]
> You should be able to use the Generic Pico 2 firmware.

it is not good, becouse board Pimoroni Pico Plus 2 using RP2350B and using more pins.

Example RPi Pico 2 using RP2350A.

**Reply by robert-hh** [reply]
Right. SO someone has to define a board or board variant. I could not tell from a first glance whether the the dies of 2350A and 2350B differ, or if it is just a different package. If it's the latter, the pins definitions for 2350 could simply be extended with the additional pins, even if they cannot be used. But that is a common situation.
